screens/Scan/SelectVidOverlay.tsx

```diff
@@ -30,7 +30,7 @@ export const SelectVidOverlay: React.FC<SelectVidOverlayProps> = (props) => {
           Share {controller.vidLabel.singular}
         </Text>
         <Text margin="0 0 16 0">
-          Choose the VID you'd like to share with{' '}
+          Choose the {controller.vidLabel} you'd like to share with{' '}
```

Comment:
this throws an error, should be `{controller.vidLabel.singular}`?

Reply:
ah yep, thanks for catching that!